.changeset/deprecation.md

```diff
@@ -0,0 +1,9 @@
+---
+"@asyncapi/generator": minor
+---
+
+- Deprecation of `ag` CLI. It is deprecated in favour of the `AsyncAPI CLI` that a single entry point for all the AsyncAPI tools. No new features will be adde to `ag` and it will be completely removed. Official documentation of AsyncAPI Generator do not mention `ag` for over a year, and only `AsyncAPI CLI` and `asyncapi generate fromTemplate` are promoted. Read [migration guide](https://www.asyncapi.com/docs/tools/generator/migration-cli) that will help you understand how to migrate your `ag` commands to the new AsyncAPI CLI.
```

Comment:
```suggestion
- **Deprecation of `ag` CLI**: The `ag` CLI is deprecated in favour of the `AsyncAPI CLI` that is a single entry point for all the AsyncAPI tools. No new features will be added to `ag` and it will be completely removed. The official documentation of AsyncAPI Generator has not mentioned `ag` for over a year, instead only using `AsyncAPI CLI` and `asyncapi generate fromTemplate` commands. Refer to the [migration guide](https://www.asyncapi.com/docs/tools/generator/migration-cli) that will help you understand how to migrate your `ag` commands to the new `AsyncAPI CLI` command.
```